iothub/device/tests/Microsoft.Azure.Devices.Client.Tests.csproj

```diff
@@ -26,6 +26,7 @@
     <Compile Remove="ClientWebSocketTransportTests.cs" />
     <Compile Remove="Mqtt\ClientWebSocketChannelTests.cs" />
     <PackageReference Include="DotNetty.Codecs.Mqtt" Version="0.7.0" />
+    <PackageReference Include="DotNetty.Common" Version="0.7.1" />
```

Comment:
q - with the test project having a reference to the source code project, don't we get all these dependencies automatically?

Reply:
I did this blindly by adding references to any assemblies that referenced DotNetty. I'm not 100% sure if the transitive dependencies overwrite the specific ones from the referenced package.